packages/fxa-auth-server/lib/senders/emails/partials/icon/index.mjml

```diff
@@ -6,17 +6,17 @@
   <% if (locals.productIconURLOld && locals.productIconURLNew) { %>
```

Comment:
What language is this? EJS?
I tried reading the mjml docs but didn't see anything obvious re:scripting languages.

Curious if there's a way we could make a [slightly gross] helper function that could determine the image dimensions from the image itself instead of hardcoding values like "58px".
I've used http://npm.im/image-size in the past with some success (albeit mostly just linting a series of logos to verify that they were all roughly 150&times;150px).

Reply:
> What language is this? EJS? I tried reading the mjml docs but didn't see anything obvious re:scripting languages.
> 
Yep, EJS. 

> Curious if there's a way we could make a [slightly gross] helper function that could determine the image dimensions from the image itself instead of hardcoding values like "58px". I've used http://npm.im/image-size in the past with some success (albeit mostly just linting a series of logos to verify that they were all roughly 150×150px).
>
Hmm, I've only used `image-size` to resize an uploaded image before it's stored and saved as a hero image/thumbnail when I was on AMO. We'll file a follow-up maintenance item for this if we can utilize it here. Thanks for the suggestion!